README.md

```diff
@@ -59,8 +59,10 @@ pet's pre-requisites.
 **Stable release:**
 
 ```shell
-$ tar zxvf pluto-0.11.4.tar.gz
-$ cd pluto-0.11.4/
+$ PLUTO_VERSION=0.12.0
+$ tar zxvf pluto-${PLUTO_VERSION}.tar.gz
+$ cd pluto-${PLUTO_VERSION}/
+$ ./autogen.sh
```

Comment:
A release package shouldn't require a user to generate/regenerate the build config. The user shouldn't even ideally need to have autoconf/automake to build pluto from a release package (as opposed to the git repo). Please drop `./autogen.sh` - not including that step was intentional.

Reply:
If I pull the latest release (`0.12.0`), then the build config (a `configure` file that I can run with `./configure`) isn't present.  This is why I added the `./autogen.sh` step.

```sh
wget https://github.com/bondhugula/pluto/releases/download/0.12.0/pluto-0.12.0.tar.gz
tar xfvz pluto-0.12.0.tar.gz 
cd pluto-0.12.0/
```